src/pages/Pokemon/Nav.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { useLocation } from "react-router-dom";
+import './pokemon.css'
+
+const Nav = () => {
+    const { state } = useLocation();
```

Comment:
I know this component it just a dummy but let's remove this `useLocation` hook anyways to make it cleaner 👍